.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+# Description of the PR
+
+<!-- Please include a summary of the change, including relevant motivation and context. -->
+
+<!-- If this PR fixes an issue, please state this using "Fixes #XYZ" -->
+
+# PR Checklist
+
+- [ ] All commits have [a Developer Certificate of Origin (DCO)](https://wiki.linuxfoundation.org/dco) -- they are generated using `-s` flag to `git commit`.
+- [ ] All new changes are covered by tests
+- [ ] If GraphQL schema is changed, `make generate` has been run
+- [ ] All code is formatted to follow project's coding style (`make format`)
```

Comment:
`golangci-lint run` or `make lint`

Reply:
Oh, `make format` is just the markdown. Should we put everything under `make format`?

Reply:
The line "All code passes linting and formatting CI" is kind of redundant with this one.

We have
`make format` -> format/wrap markdown files
`make fmt` -> gofmt / goimports / copyright
`make lint` -> golangci-lint.

Maybe need one for all of them?

Reply:
Yeah, let's make `make format` run all 3 of them

Reply:
Oh, `make fmt` and `make lint` are actually only checking if the formatting is valid or erroring out.

Reply:
Let's leave putting these together for another time. Removed duplciation for now